kairon/shared/data/customer_order_processor.py

```
@@ -145,9 +208,46 @@ def create_order(bot: Text, sender_id: str, persona_type: Optional[str], order_p
+            try:
+                razorpay_resp = CustomerOrderProcessor._create_razorpay_payment_link(
+                    api_key, api_secret, order_id, order_payload, callback_url
+                )
+                payment_id = razorpay_resp.get("id", "")
+                payment_link = razorpay_resp.get("short_url", "")
+                OrderDetails.objects(id=order.id).update_one(
+                    set__additional_info={"payment_id": payment_id, "payment_link": payment_link}
+                )
+            except AppException:
+                raise
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)